spec/services/record_declaration_spec.rb

```diff
@@ -418,6 +430,26 @@
       it_behaves_like "creates a participant declaration"
       it_behaves_like "creates participant declaration attempt"
       it_behaves_like "checks for mentor completion event"
+
+      context "when the declaration_type is retained" do
```

Comment:
can we add a spec for started or completed to show they go through? (unless that's already tested above with the flag